apps/docs/scripts/components-with-animation.js

```diff
@@ -1,16 +1,16 @@
 /* This file is generated by a script. Do not modify. */
 
 export const supportedComponents = [
-  'Button',
   'Box',
+  'Button',
   'ComparisonTable',
   'FAQ',
   'Heading',
   'Image',
   'Label',
-  'River',
```

Comment:
This happened automatically on build so I assume this is ok to push otherwise we will have to keep reverting it.